elb_log_ingestor/main.py

```diff
@@ -0,0 +1,75 @@
+import http.server
+import os
+import pathlib
+import queue
+import sys
+import threading
+
+import boto
+import elasticsearch
+
+import api_endpoint
+import elasticsearch_shipper
+import elb_log_fetcher
+import elb_log_parse
+import stats
+
+
+def start_server():
```

Comment:
It would be good to implement a `stop_server()` function that would gracefully shut down the threads to prevent data loss. Just a suggestion, not a requirement. :)